app/controllers/items_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ItemsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :redirect_to_root_path, only: [:edit, :update]
```

Comment:
```suggestion
  before_action :redirect_to_root_path, only: [:edit, :update,:destroy]
```
trelloの実装要件に`出品者だけが商品情報を削除できること`とあるように削除機能でもリダイレクトされるように追加しましょう。
こうすることで出品者だけが削除できるように指定できます。